firmware/boron-ins-fsm/src/BraveSensorProductionFirmware.ino

```diff
@@ -11,7 +11,7 @@
 #include "tpl5010watchdog.h"
```

Comment:
Here's an example of a past PR that has an entry in  the `CHANGELOG` and an explanation of what was tested in the description of the PR: #217 . These are part of the minimum standards for pull requests. 

We also always add the ClickUp Task ID(s) to the title of the PR because there is a bot that will automatically then add a link to the Task in the PR's comments. I'll clarify the documentation to make that part clearer.